exasol_script_languages_developer_sandbox/lib/github_release_access.py

```diff
@@ -0,0 +1,60 @@
+from pathlib import Path
+
+from github import Github, GithubException
+from github.Repository import Repository
+
+from exasol_script_languages_developer_sandbox.lib.logging import get_status_logger, LogType
+
+LOG = get_status_logger(LogType.RELEASE_ACCESS)
+
+
+class GithubReleaseAccess:
+
+    def __init__(self, gh_token: str):
+        self._gh_token = gh_token
+
+    def create_release(self, branch: str, title: str) -> int:
+        """
+        Implements creation of a Github Draft Release.
+        See https://docs.github.com/en/rest/releases/releases for details.
+        Returns the internal ID of the new release.
+        """
+        release = self._get_repo.create_git_release(tag="", name=title, message="Test-Release",
+                                                    draft=True, prerelease=True, target_commitish=branch)
+        return release.id
+
+    def update_release_message(self, release_id: int, message_to_append: str) -> None:
+        """
+        Updates an existing release, identified by parameter "release_id".
+        The release message body will be extended by parameter "message_to_append".
+        See https://docs.github.com/en/rest/releases/releases for details.
+        """
+        release = self._get_repo.get_release(release_id)
+        body = release.body
+        body += message_to_append
+        release.update_release(release.title, body)
+
+    def upload(self, archive_path: str, label: str, release_id: int, content_type: str):
```

Comment:
Do we have assets to upload? Maybe the checksums of the inages

Reply:
I upload currently the links of the assets as markdown file